.eslintrc.json

```diff
@@ -15,9 +20,24 @@
         "sourceType": "module"
     },
     "plugins": [
-        "react"
+        "react",
+        "react-hooks",
+        "jest",
+        "cypress"
     ],
     "rules": {
+        "react-hooks/rules-of-hooks": "error", // Checks rules of Hooksyarn run eslint --init
```

Comment:
Copy/paste error in the comment ? :smile:

Reply:
Fixed :)